frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -88,26 +95,6 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
             <LemonDivider />
             <SurveyReleaseSummary id={id} survey={survey} targetingFlagFilters={targetingFlagFilters} />
             <LemonDivider />
-            <div className="flex items-center gap-2 justify-end">
-                <LemonButton
-                    data-attr="cancel-survey"
-                    type="secondary"
-                    loading={surveyLoading}
-                    onClick={() => {
-                        if (isEditingSurvey) {
-                            editingSurvey(false)
-                            loadSurvey()
-                        } else {
-                            router.actions.push(urls.surveys())
-                        }
-                    }}
-                >
-                    Cancel
-                </LemonButton>
-                <LemonButton type="primary" data-attr="save-survey" htmlType="submit" loading={surveyLoading}>
-                    {id === 'new' ? 'Save as draft' : 'Save'}
-                </LemonButton>
-            </div>
```

Comment:
Drive-by change that's not really in scope, but Boy Scout Principle and all that. I noticed that we had a footer on this page, which felt superfluous given that the page has a sticky header.  Here's what it looks liek with the footer

<img width="1264" alt="Screenshot 2024-05-20 at 5 07 21 PM" src="https://github.com/PostHog/posthog/assets/4853149/099b101c-9c71-4a4f-a272-b1413f965e6a">

And what it looks like without

<img width="1217" alt="Screenshot 2024-05-20 at 5 06 59 PM" src="https://github.com/PostHog/posthog/assets/4853149/698170f1-3e96-4400-b393-94d1bf320d02">

I prefer without.